config/routes.rb

```diff
@@ -2,11 +2,15 @@
   root 'homes#index'
   devise_for :users
 
-  resources :stops, only: [:index, :show, :new, :create]
+  resources :stops, only: [:index, :show, :new, :create] do
+    resources :reviews, only: [:new, :create]
```

Comment:
reviews should be created in react. You arent or shouldnt be having paths for reviews here, only in the API controllers